tobac/utils.py

```diff
@@ -140,32 +218,65 @@ def mask_cell_surface(mask, cell, track, masked=False, z_coord="model_level_numb
 
 
 def mask_cell_columns(mask, cell, track, masked=False, z_coord="model_level_number"):
-    """Create mask with entire columns for individual cell
-    Input:
-    mask:              iris.cube.Cube
-                       cube containing mask (int id for tacked volumes 0 everywhere else)
-    Output:
-    variable_cube_out: iris.cube.Cube
-                       Masked cube for untracked volume
+    """Create mask with entire columns for individual cell.
+
+    Parameters
+    ----------
+    mask : iris.cube.Cube
+        Cube containing mask (int id for tacked volumes 0 everywhere
+        else).
+
+    cell : int
+        Interger id of cell to create the masked cube for.
+
+    track : pandas.DataFrame
+        Output of the linking.
+
+    masked : bool, optional
+        Bool determining whether to mask the mask for the cell where
+        it is 0. Default is False.
+
+    z_coord : str, optional
+        Default is 'model_level_number'.
+
+    Returns
+    -------
+    mask_i : iris.cube.Cube
+        Masked cube for untracked volume.
+
+    Notes
+    -------
+    Function is not working since mask_features_columns()
```

Comment:
I think that is a good idea @freemansw1. Seems like we have a couple of non-working functions and raising an error would help users and developers to quickly identify those instead of looking for the bug in their code. So let us start with this one here. The only thing I am wondering is how we decide whether to deprecate and clean up in the longer term or if the function is worth raising an issue so that we can make it work again.

Reply:
I added an `NotImplementedError` to the function. These could serve es a todo-indicator for funtions that don't work atm.